modules/core/src/main/scala/org/legogroup/woof/Logger.scala

```diff
@@ -29,7 +29,9 @@ open class Logger[F[_]: StringLocal: Monad: Clock](output: Output[F], outputs: O
       message: String,
       context: List[(String, String)],
   ): F[String] =
-    Clock[F].realTimeInstant.map(now => summon[Printer].toPrint(now, level, info, message, context))
+    Clock[F].realTime
+      .map(d => Instant.ofEpochMilli(d.toMillis))
+      .map(now => summon[Printer].toPrint(now, level, info, message, context))
```

Comment:
As mentioned above, Scala.js doesn't include support for `java.time.Instant` without adding additional dependencies. So Cats Effect does not use it in its Scala.js APIs.